swift-sdk/IterableInApp.swift

```diff
@@ -0,0 +1,125 @@
+//
+//
+//  Created by Tapash Majumder on 11/5/18.
+//  Copyright © 2018 Iterable. All rights reserved.
+//
+
+import Foundation
+
+/// `show` to show the inApp otherwise `skip` to skip.
+@objc public enum ShowInApp : Int {
+    case show
+    case skip
+}
+
+@objc
+public protocol IterableInAppManagerProtocol {
+    /// - returns: A list of all messages
+    @objc(getMessages) func getMessages() -> [IterableInAppMessage]
+
+    /// - parameter message: The message to show.
+    /// - parameter consume: Set to true to consume the event from the server queue if the message is shown. This should be default.
+    /// - parameter callback: block of code to execute once the user clicks on a link or button in the inApp notification.
+    ///   Note that this callback is called in addition to calling `IterableCustomActionDelegate` or `IterableUrlDelegate` on the button action.
+    @objc(showContent:consume:callbackBlock:) func show(message: IterableInAppMessage, consume: Bool, callback:ITEActionBlock?)
```

Comment:
`showContent` should be `showMessage` here I guess; there are 1 or 2 places elsewhere where `show(contentL:` or `showContent` is mentioned.